Meta/serenity.sh

```diff
@@ -300,7 +300,7 @@ ensure_toolchain() {
     if [ "$TOOLCHAIN_TYPE" = "GNU" ]; then
         local ld_version
         ld_version="$("$TOOLCHAIN_DIR"/bin/"$TARGET"-pc-serenity-ld -v)"
-        local expected_version="GNU ld (GNU Binutils) 2.38"
+        local expected_version="GNU ld (GNU Binutils) 2.39"
         if [ "$ld_version" != "$expected_version" ]; then
             echo "Your toolchain has an old version of binutils installed."
             echo "    installed version: \"$ld_version\""
```

Comment:
Do you think we should keep this check?

I originally added it in #12425, because I made a change to the linker flags that would make compilation with an older binutils fail. I couldn't manage to make CMake check the linker's version, so I added the check here instead. Here, we don't have this problem, because we're updating it alongside gcc, so CMake will force a toolchain rebuild.

I guess we might have a similar situation in the future; if we want to keep this code around, could you remove the FIXME?

Reply:
Yea, I think it makes sense to keep it. FIXME removed.